tests/IssueDetailsTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace tests;
+
+use ITBugTracking\Entities\IssueDetails;
+use PHPUnit\Framework\TestCase;
+use TypeError;
+
+class IssueDetailsTest extends TestCase
+{
+    public function testIssueDetails_JsonSerialize_success()
+    {
+        $issueDetails = new IssueDetails();
+
+        $issueDetails->id = 5;
+        $issueDetails->title = "A Title";
+        $issueDetails->description = "This is a description.";
+        $issueDetails->severity = "Severe";
+        $issueDetails->date_created = "05/11/2024";
+        $issueDetails->comment_count = 5;
+        $issueDetails->reporter = "Joe Blogs";
+        $issueDetails->department = 1;
+        $issueDetails->comments = ["comment one", "comment two", "comment three"];
+
+        $expected = json_encode([
+            'id' => 5,
+            'title' => "A Title",
+            'severity' => "Severe",
+            'date_created' => "05/11/2024",
+            'comment_count' => 5,
+            'reporter' => "Joe Blogs",
+            'department' => 1,
+            'description' => "This is a description.",
+            'comments' => ["comment one", "comment two", "comment three"]
+        ]);
+
+        $actual = json_encode($issueDetails);
+
+        $this->assertEquals($expected, $actual);
+    }
+
+    public function testIssueDetails_JsonSerialize_NullComments_success()
```

Comment:
Not null comments, that will confuse people.

No comments.

Renaming the tests so that the names are still accurate should be part of making functionality changes to the tests.

Reply:
Jordan's done this